ui/components/institutional/institutional-entity-done-page/institutional-entity-done-page.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import Button from '../../ui/button';
+import { getMostRecentOverviewPage } from '../../../ducks/history/history';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { Text } from '../../component-library';
+import {
+  TextColor,
+  BorderRadius,
+  TypographyVariant,
+} from '../../../helpers/constants/design-system';
+import Box from '../../ui/box';
+
+export default function InstitutionalEntityDonePage(props) {
+  const mostRecentOverviewPage = useSelector(getMostRecentOverviewPage);
+  const t = useI18nContext();
+  const { history, location } = props;
+  const { state } = location;
+
+  return (
+    <Box
+      className="page-container institutional-entity-done__container"
+      borderRadius={BorderRadius.none}
```

Comment:
You should not *need* borderRadius  prop AND the styles on this class, if borderRadius set to none does not work let me know otherwise remove the border-radius style.

Reply:
Yes, you are right, removing the className